cmd/earthly/build_cmd.go

```diff
@@ -558,6 +567,44 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 	return nil
 }
 
+func (app *earthlyApp) getOrgForAnalytics() string {
```

Comment:
From [`Effective Go`](https://go.dev/doc/effective_go#Getters):

> it's neither idiomatic nor necessary to put Get into the getter's name.

I also think `analyticsOrg` reads better than `orgForAnalytics`.
```suggestion
func (app *earthlyApp) analyticsOrg() string {
```